modules/internal/fixInternalDocs.sh

```diff
@@ -217,15 +217,6 @@ replace "chpl_bytes" "bytes" $file
 
 ## End of Bytes ##
 
-## ChapelEnv ##
-
-file="./ChapelEnv.rst"
-fixTitle "Chapel Environment Variables" $file
-replace " = AppendExpr.Call09" "" $file
```

Comment:
I suspect this has probably been fixed already, but it would be good to check for any remaining instances in the generated docs

Reply:
There are none in ChapelEnv itself (used split init to fix them) but I made a future about remaining odd output from chpldoc in #16866 / #16867